src/gui/Src/Gui/MainWindow.cpp

```diff
@@ -338,6 +338,8 @@ MainWindow::MainWindow(QWidget* parent)
 
     connect(mCpuWidget->getStackWidget(), SIGNAL(displayReferencesWidget()), this, SLOT(displayReferencesWidget()));
 
+    connect((QObject*)mCpuWidget->getInfoBoxWidget(), SIGNAL(displayReferencesWidget()), this, SLOT(displayReferencesWidget()));

```

Comment:
The compiler complains about this being without a cast
![image](https://cloud.githubusercontent.com/assets/13303069/25562178/7695e7ae-2d4c-11e7-9c7c-d0fa136f7157.png)